main/user_main.c

```diff
@@ -314,8 +411,13 @@ void user_init(void) {
 	                  MAX_CONNECTIONS,
 	                  HTTPD_FLAG_NONE);
 	httpdFreertosStart(&httpdFreertosInstance);
-
-	init_wifi(true); // Supply false for STA mode
+	
+	
+	//WIFI_MODE_NULL = 0,  /**< null mode */
```

Comment:
can you remove these commented out lines in this PR?

Reply:
That's actually my lazy documentation ;-)  
Will revert that WiFi stuff anyway.